__tests__/app.test.js

```diff
@@ -467,4 +465,50 @@ describe("PATCH /api/articles/:articleid", () => {
   });
 });
 
+describe("DELETE api/comments/:comment_id", () => {
+  test("should return 200", () => {
+    return request(app).delete("/api/comments/1")
+      .expect(204)
+  });
+  test("should delete the comment of the associated comment id", () => {
+    const articleId = 9;
+    const count = commentData.reduce((counter, obj) => {
+      if (obj.article_id === articleId) counter += 1;
+      return counter;
+    }, 0);
+    return request(app).delete("/api/comments/1")
+      .expect(204)
+      .then(() => {
+        return request(app)
+          .get(`/api/articles/${articleId}/comments`)
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.comments.length).toEqual(count-1);
+          });
+      });
+  });
+  test("should return an empty object  ", () => {
```

Comment:
Delete should not return anything. In case of a 204 status code, Express doesn't send back anything apart from the status code. The object you see there is from Supertest and we don't test for that.